index.rst

```diff
@@ -10,20 +10,849 @@
+
+Tier 2:
+^^^^^^^
+The following are in order of importance, but again are largely parallelizable between various parties.
+
+- Setup and configure the Camera Diagnostic Cluster.
+  This includes running and publishing the low level diagnostics, then progressing on the alert infrastructure.
+-  Deploy the Rapid Analysis framework at the summit
+   Initial efforts should be focused on development of interfaces and not speed.
+   Capabilities of each part can be expanded incrementally.
+   Early testing can store metric values in the Butler.
+- Create a database in Sasquatch for recording the Rapid Analysis Metrics
+- Define alert framework for the alarming metrics and for when processed images are available
+
+Once the above are completed, then the following can be performed:
+
+- Record Rapid Analysis Metrics in the Summit Sasquatch instance
+- Replicate Rapid Analysis Metrics from the Summit Sasquatch instance to the USDF Sasquatch instance
+- Start issuing alerts that metrics and processed images are available
+- Create visualizations w/ Chronograf, etc.
+
+Tier 3:
+^^^^^^^
+
+- Development of the Catcher
+- Performing daily `12-24 hours`_ SFP at USDF.
+  This capability is not required to support realtime decisions during nighttime operations.
+  There is requirement to wait until the morning to begin reductions and in fact processing the results as the data streams in is preferred.
+  Note that early runs can put data into the butler and can then be expanded to a database.
+- CVT augmented to read processed images from Rapid Analysis, then expanded to support any full frame image persisted in the Butler
+- Create templates for development of Catcher, Bokeh, and possibly LOVE displays by SIT-Com personnel and/or project software developers.
+- Develop training examples.
+  Ideally, this will be performed in conjunction with the development of templates.
+
+Again, developing a common toolset between the commissioning team and the DRP, or one that is based off the tooling being discussed here, is strongly recommended.
+This is not explicitly listed as a priority as it should be a continually ongoing activity.
+
+Recommended Tools
+^^^^^^^^^^^^^^^^^
+
+Once the frameworks defined above and prior to entering commissioning, a series of additional tools need to be constructed to facilitate commissioning.
+The following is a non-exhaustive list of general tools that will be required and are not currently owned by any person and/or group.
+It does not include subsystem specific displays such as what will be required for commissioning the Active Optics System.
+
+#. An on-the-fly telescope offset calculation and implementation tool.
+#. A tool to display scalar fields, as discussed in `Deliverable 4`_.
+#. A display showing the calculated metrics for each image, with indicators when values are out of range.
+   The contents should be linked to down-range diagnostic tools/displays that are accessed upon "clicking."
```

Comment:
"down-range" ?

Reply:
ya, I guess that's slang... i removed it.